### PR TITLE
Make ORD Aggregator deployment

### PR DESCRIPTION
**Pull request by kaloyanspiridonov**
**Description**

Changes proposed in this pull request:
- ...

**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] Mocks are regenerated, using the automated script

### CODE REVIEW

**Review by alextargov — DISMISSED** [review]

File: chart/compass/charts/ord-aggregator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+reviewers:
```
Comment (PetarTodorovv) [inline chart/compass/charts/ord-aggregator/OWNERS:1]: You need to adapt the CODEOWNERS file as well

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Comment by alextargov** [comment]
reset-vm

**Review by alextargov — DISMISSED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**